dunwoody_disaster/CharacterFactory.py

```diff
@@ -252,28 +254,33 @@ def createTestChar() -> Character:
         character = CharacterFactory.createCharacter("Test-Char", "blank")
         return character
 
+    @staticmethod
+    def saveCharacter(character: Character) -> None:
+        """
+        Saves a character to a json file
+        :param character: The character to save
+        """
+        with open(f"{character.name}.json", "w") as f:
+            json.dump(character.serialize(), f)
```

Comment:
Can the user play as multiple characters? I.e. load different save files? If so we should put them in a `savedata` or `storage` directory, to not clutter up the working directory. If not we don't need to dynamically named save file.